documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -76,217 +73,135 @@ controlPlane:
+
 ifdef::watermark[]
-or `http://[2620:52:0:1307::1]/rhcos-<version>-qemu.x86_64.qcow2.gz?sha256=<uncompressed_sha256>`.
-endif::[]
 
+ or `http://[2620:52:0:1307::1]/rhcos-<version>-qemu.x86_64.qcow2.gz?sha256=<uncompressed_sha256>`.
+
+endif::[]
 
 | `clusterOSImage`
 |
-| A URL to override the default operating system for cluster nodes. The URL must include a sha256 hash of the image. Example https://mirror.openshift.com/images/rhcos-<version>-openstack.qcow2.gz?sha256=<compressed_sha256>
+| A URL to override the default operating system for cluster nodes. The URL must include a SHA-256 hash of the image. For example,  https://mirror.openshift.com/images/rhcos-<version>-openstack.qcow2.gz?sha256=<compressed_sha256>.
+
+
+| `provisioningNetwork`
+|
+| Set this parameter to `Disabled` to disable the requirement for a `provisioning` network.
+
+| `provisioningHostingIp`
+|
+| Set this parameter to an available IP address on the `baremetal` network when the `provisioningNetwork` configuration setting is set to `Disabled`.
 
 |===
 
 [id="hoststable"]
 .Hosts
 
-The `hosts` parameter is a list of separate bare metal assets that
-should be used to build the cluster.
+The `hosts` parameter is a list of separate bare metal assets used to build the cluster.
 
 |===
 |Name |Default |Description
 | [[name]]`name` |  | The name of the BareMetalHost resource to associate with the details. e.g. `openshift-master-0`
-| [[role]]`role` |  | Either `master` or `worker`.
-| `bmc` | | Connection details for the baseboard management controller. xref:bmcadressing[See below for details.]
-| [[bootMACAddress]]`bootMACAddress` |  | The MAC address of the NIC the host will use to boot on the provisioning network.
-| [[hardwareProfile]]`hardwareProfile` | `default`|This parameter exposes the device name that the installer attempts to deploy the {product-title} for the master and worker nodes. The value defaults to `default` for masters and `unknown` for workers. The list of profiles includes: `default`, `libvirt`, `dell`, `dell-raid`, `openstack`. The `default` parameter attempts to install on `/dev/sda` of your cluster node.
+| [[role]]`role` |  | The role of the bare metal node. Either `master` or `worker`.
+| `bmc` | | Connection details for the baseboard management controller. See the xref:bmcadressing[BMC addressing section] for additional details
+| [[bootMACAddress]]`bootMACAddress` |  | The MAC address of the NIC the host will use to boot on the `provisioning`  network.
+| [[hardwareProfile]]`hardwareProfile` | `default`|This parameter exposes the device name that the installer attempts to deploy the {product-title} cluster for the control plane and worker nodes. The value defaults to `default` for control plane nodes and `unknown` for worker nodes. The list of profiles includes: `default`, `libvirt`, `dell`, `dell-raid`, and `openstack`. The `default` parameter attempts to install on `/dev/sda` of the {product-title} cluster nodes.
```

Comment:
hardwareProfile is no longer required with 4.5+

Reply:
so < 4.5 right?

Reply:
yes